backend/src/pdf_gen/typst_tests.rs

```diff
@@ -47,30 +47,22 @@ fn random_value<T: Copy>(rng: &mut impl rand::Rng, variants: &[T]) -> T {
     *variants.choose(rng).unwrap()
 }
 
-fn random_date(rng: &mut impl rand::Rng) -> NaiveDate {
-    NaiveDate::from_ymd_opt(
-        rng.random_range(1900..=2100),
-        rng.random_range(1..=12),
-        rng.random_range(1..=28),
-    )
-    .unwrap()
+fn random_date_time(rng: &mut impl rand::Rng) -> DateTime<Local> {
+    let now = Local::now().timestamp();
+    let ten_years = 10 * 365 * 24 * 60 * 60;
+    let secs = rng.random_range(now..(now + ten_years));
```

Comment:
We should also generate dates in the past because those would normally happen pretty frequently when using abacus